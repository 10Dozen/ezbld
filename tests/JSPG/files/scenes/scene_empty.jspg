# EmptyScene
# EmptyScene2
# AnotherEmpty