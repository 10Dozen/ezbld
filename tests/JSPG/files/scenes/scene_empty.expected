[0]
cls = JSPGScene
name = EmptyScene | TEXT

[1]
cls = JSPGScene
name = EmptyScene2 | TEXT

[2]
cls = JSPGScene
name = AnotherEmpty | TEXT